.changes/next-release/feature-AWSSDKforJavav2-d08c3c0.json

```diff
@@ -0,0 +1,5 @@
+{
+    "category": "AWS SDK for Java v2", 
+    "type": "feature", 
+    "description": "Automatically retry streaming downloads to a file if they fail or are interrupted."
```

Comment:
Once I've got the PR open I've been trying to remember to add the PR link to the change-log entry.

Reply:
Should we have some way of automating that so that it's formatted consistently?